src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2611,6 +2611,72 @@ Task Synchronization API can be used for multiple dynamic synchronization patter
 
 A complete example of Producer / Consumer, _ServiceDeployerConsumer_ can be downloaded link:examples/ServiceDeployerConsumer.xml[here].
 
+[[_task_signal_api]]
+=== Signal API
+
+The *Signal API* is a high-level API that enables users to interact with running jobs via signals (we also call them actions).
+That is, it allows a job to expose specific actions (e.g., soft stop, scale up, scale down) that can triggered by users from the *Workflow Execution portal*.
```

Comment:
what does mean soft stop ?

Reply:
also what does mean scale up a job ?

Reply:
Soft stop i
I did not say 'scale up a job', scale up and down are examples of **actions**. 
Soft stop is also an example of actions that means properly finishing a job (instead of executing kill in the scheduling portal or REST API)

Reply:
ok but I believe it important to well define these actions in the doc. I mean explain explicitly what is a soft job and what to scale up and down

Reply:
I agree with @mboussaa, this description is not clear. The purpose of the API is not to handle actions (which is HOW the workflow react to a signal), but to expose signals.

Reply:
Considered this comment and updated the doc